tests/integration/security/authz/authz_test.go

```diff
@@ -0,0 +1,1547 @@
+//go:build integ
+// +build integ
+
+// Copyright Istio Authors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package security
+
+import (
```

Comment:
In most of the scenarios you're using the conditions with `notValues`. Shouldn't we try to check all the attributes which is there in authz policy (eg. https://istio.io/latest/docs/reference/config/security/authorization-policy/#Operation). I know under the hood it might use the same stuff (like conditions) but if anything gets changed with any of these attributes then tests based on this will help us analyze that in future.